src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4743,10 +4750,10 @@ public override bool PreVisit(CallNode node)
                     return false;
                 }
 
-                // If there are no overloads with lambdas, we can continue the visitation and
+                // If there are no overloads with lambdas or identifiers, we can continue the visitation and
                 // yield to the normal overload resolution.
-                var overloadsWithLambdas = overloads.Where(func => func.HasLambdas);
-                if (!overloadsWithLambdas.Any())
+                var overloadsWithLambdasOrIdentifiers = overloads.Where(func => func.HasLambdas || func.HasIdentifiers);
```

Comment:
> || func.HasIdentifiers [](http://example.com/codeflow?start=95&length=23)

Need to triple check the impact of this. 

func.HasIdentifiers will be true even if the feature flag is not enabled - so this will impact all code paths (bindings in Canvas apps)

Reply:
You're right, this isn't correct.
I'll add feature flag consideration here.

Reply:
I'll test with Power Apps later